ipi/inputs/forcefields.py

```diff
@@ -207,6 +207,37 @@ def fetch(self):
             raise ValueError("Negative timeout parameter specified.")
 
 
+class InputFFQUIP(InputForceField):
+
+    attribs = {}
+    attribs.update(InputForceField.attribs)
+
+    default_help = """A general QUIP interaction potential evaluator.
```

Comment:
Use fields (see e.g. plumedff) to read these options, rather than stuffing everyting into params